### PR TITLE
[Merged by Bors] - feat(AlgebraicGeometry/ProjectiveSpectrum/Scheme): calculate stalk of `toSpec` map

### PR DESCRIPTION
**Pull request by jjaassoonn**
In this PR, we show the newly defined map of locally ringed space`toSpec` agree with the `ProjIsoSpecTopComponent.toSpec` and calculate the stalk map.

Co-authored-by: Andrew Yang <the.erd.one@gmail.com>

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by erdOne** [comment]
Thanks!
LGTM but I did a significant portion of these so I'll wait for someone else to review.

File: Mathlib/AlgebraicGeometry/ProjectiveSpectrum/Scheme.lean [hunk 1]
```diff
@@ -644,6 +644,57 @@ def toSpec (f) : (Proj| pbo f) ⟶ Spec (A⁰_ f) :=
...
+  convert (ProjIsoSpecTopComponent.ToSpec.preimage_basicOpen f t)
+  exact funext fun _ => toSpec_base_apply_eq _ _
+
+@[reassoc]
+lemma toOpen_toSpec_val_c_app (f) (U) :
+    StructureSheaf.toOpen (A⁰_ f) U.unop ≫ (toSpec 𝒜 f).val.c.app U =
+      awayToΓ 𝒜 f ≫ (Proj| pbo f).presheaf.map (homOfLE le_top).op :=
+  Eq.trans (by congr) <| ΓSpec.toOpen_comp_locallyRingedSpaceAdjunction_homEquiv_app _ U
+
+lemma toStalk_stalkMap_toSpec (f) (x) :
```
Comment (joelriou) [inline Mathlib/AlgebraicGeometry/ProjectiveSpectrum/Scheme.lean:689]:
```suggestion
@[reassoc]
lemma toStalk_stalkMap_toSpec (f) (x) :
```

**Comment by joelriou** [comment]
Thanks!

bors d+

**Pull request closed without merging**